Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/lathe.yml

```diff
@@ -299,6 +300,10 @@
       - CableStack
       - CableMVStack
       - CableHVStack
+      - ShipyardRCD # Frontier
+      - ShipyardRCDAmmo # Frontier
+      - WelderExperimental # Frontier
```

Comment:
Something that might be a nice-to-have would be to put the industrial version of the welder in the static recipes.